### PR TITLE
Dark mode: fix toggle-buttons

### PR DESCRIPTION
**Pull request by louismaximepiton**
- New `.btn-toggle` specific Boosted class

### Live preview

- https://deploy-preview-2365--boosted.netlify.app/docs/5.3/dark-mode/#toggle-buttons

### CODE REVIEW

**Comment by julien-deramond** [comment]
Some elements have a black color when the dark theme is set to the container in light mode instead of white:

Screenshot 2023-11-17 at 14 44 04

File: scss/_variables.scss [hunk 1]
```diff
@@ -1204,7 +1204,7 @@ $form-check-input-indeterminate-bg-image:       url("data:image/svg+xml,<svg xml
 
 $form-check-input-disabled-opacity:        null !default;
 $form-check-label-disabled-opacity:        null !default;
-$form-check-btn-check-disabled-opacity:    null !default;
+// Boosted mod: no $form-check-btn-check-disabled-opacity
```
Comment (julien-deramond) [inline scss/_variables.scss:1207]: Can't we keep this Sass var and apply it even if `null` just to ensure compatibility with Bootstrap like it's done for `$form-check-label-disabled-opacity` for instance?

File: scss/_buttons.scss [hunk 2]
```diff
@@ -58,8 +58,7 @@
 
   // Boosted mod: no .btn-check:focus-visible + &
 
-  // Boosted mod: custom styles for .btn-check
-  // Boosted mod: simpler rule
+  // Boosted mod: selectors are slightlty different to handle `.btn-check:checked`
```
Comment (louismaximepiton) [inline scss/_buttons.scss:61]:
```suggestion
  // Boosted mod: selectors are slightly different to handle `.btn-check:checked`
```

File: scss/_buttons.scss [hunk 3]
```diff
@@ -73,7 +72,9 @@
     // Boosted mod: no box shadow modification for the active state
     // Boosted mod: no &:focus-visible
   }
+  // End mod
 
+  // Boosted mod: selectors are slightlty different to handle `.btn-check`
```
Comment (louismaximepiton) [inline scss/_buttons.scss:77]:
```suggestion
  // Boosted mod: selectors are slightly different to handle `.btn-check`
```

**Pull request merged**